Podfile

```diff
@@ -14,9 +14,12 @@ target 'BayPass' do
   pod 'SwiftSoup'
   pod 'Stripe'
   pod 'Disk'
- 
+  pod 'Firebase/Core'
```

Comment:
As you can see in the pod file.lock Google automatically adds a whole bunch of dependencies. Is it possible to only add the Firestore pod?

Reply:
I'll try

Reply:
find the reason for using pod 'Firebase/Core'. We need to configure FirebaseApp to use Firestore database, and this config requires pod Firebase/Core

Reply:
Okay 😕 They seem to add a lot of boiler plate, but looks like we can't have any influence on that then